components/Editor/index.tsx

```diff
@@ -358,13 +419,21 @@ const Editor = ({ readOnly = false }: Props) => {
                   classNamePrefix="select"
                   menuPlacement="auto"
                 />
+
+                <Button
+                  onClick={handleCopyPermalink}
+                  transparent
+                  padded={false}
```

Comment:
I'd add a tooltip here to make it clear what it does, just pass in tooltip props as in the debugging step button icons

Reply:
Good idea, done